plugins/woocommerce/i18n/states.php

```diff
@@ -409,6 +409,32 @@
 		'DZ-48' => __( 'Relizane', 'woocommerce' ),
 	),
 	'EE' => array(),
+	'EC' => array( // Ecuador states.
+		'EC-A' => __( 'Azuay', 'woocommerce' ),
+		'EC-B' => __( 'Bolívar', 'woocommerce' ),
+		'EC-F' => __( 'Cañar', 'woocommerce' ),
+		'EC-C' => __( 'Carchi', 'woocommerce' ),
+		'EC-H' => __( 'Chimborazo', 'woocommerce' ),
+		'EC-X' => __( 'Cotopaxi', 'woocommerce' ),
+		'EC-O' => __( 'El Oro', 'woocommerce' ),
+		'EC-E' => __( 'Esmeraldas', 'woocommerce' ),
+		'EC-W' => __( 'Galápagos', 'woocommerce' ),
+		'EC-G' => __( 'Guayas', 'woocommerce' ),
+		'EC-I' => __( 'Imbabura', 'woocommerce' ),
+		'EC-L' => __( 'Loja', 'woocommerce' ),
+		'EC-R' => __( 'Los Ríos', 'woocommerce' ),
+		'EC-M' => __( 'Manabí', 'woocommerce' ),
+		'EC-S' => __( 'Morona Santiago', 'woocommerce' ),
+		'EC-N' => __( 'Napo', 'woocommerce' ),
+		'EC-D' => __( 'Orellana', 'woocommerce' ),
+		'EC-Y' => __( 'Pastaza', 'woocommerce' ),
+		'EC-P' => __( 'Pichincha', 'woocommerce' ),
+		'EC-SE' => __( 'Santa Elena', 'woocommerce' ),
+		'EC-SD' => __( 'Santo Domingo de los Tsáchilas', 'woocommerce' ),
+		'EC-U' => __( 'Sucumbíos', 'woocommerce' ),
+		'EC-T' => __( 'Tungurahua', 'woocommerce' ),
+		'EC-Z' => __( 'Zamora Chinchipe', 'woocommerce' ),
```

Comment:
Please hyphenate this to conform to the Unicode CLDR standard. https://github.com/unicode-org/cldr/blob/release-40/common/subdivisions/en.xml#L1251